src/acp/control.ts

```diff
@@ -528,3 +528,52 @@ export function removeMemberPolicyUrlAll<ResourceExt extends WithAccessibleAcr>(
   );
   return updatedResource;
 }
+
+/**
+ * Gets a human-readable representation of the given [[Control]] to aid debugging.
+ *
+ * Note that changes to the exact format of the return value are not considered a breaking change;
+ * it is intended to aid in debugging, not as a serialisation method that can be reliably parsed.
+ *
+ * @param control The Control to get a human-readable representation of.
+ */
+export function acrAsMarkdown(
+  resourceWithAcr: WithResourceInfo & WithAccessibleAcr
+): string {
+  let markdown = `## Access control for ${getSourceUrl(resourceWithAcr)}\n`;
```

Comment:
Should not this be a `h1`, and not a `h2` ?

Reply:
Oh that's a good point: acc0ac6fecf791f3f0edad57fb9890ba5fd64329